src/txmempool.cpp

```diff
@@ -1580,13 +1580,6 @@ CFeeRate CTxMemPool::GetMinFee(size_t sizelimit) const {
     return std::max(CFeeRate(rollingMinimumFeeRate), incrementalRelayFee);
 }
 
-void CTxMemPool::UpdateMinFee(const CFeeRate& _minReasonableRelayFee)
```

Comment:
Should also drop it from txmempool.h

Reply:
https://github.com/PastaPastaPasta/dash/commit/2ce335ef92fc32f049d18fae976139bbc68bde32